WordPress/src/main/java/org/wordpress/android/designsystem/DesignSystemActivityCompose.kt

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.designsystem
+
+import android.content.res.Configuration
+import android.os.Bundle
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import org.wordpress.android.ui.LocaleAwareActivity
+import org.wordpress.android.util.extensions.setContent
+
+class DesignSystemActivityCompose : LocaleAwareActivity() {
```

Comment:
Probably `Compose` suffix is redundant here.  Just `DesignSystemActivity` would do

Reply:
Agreed, I initially named it with `Compose` since I had another DesignSystemActivity that wasn't using Compose 😅  I'll update this